src/pages/categories.js

```diff
@@ -40,29 +40,28 @@ const descriptions = {
 	Staking: 'Protocols that allow you to stake assets in exchange of a reward',
 	Services: 'Protocols that provide a service to the user',
 	'Yield Aggregator': 'Protocols that aggregated yield from diverse protocols',
-	Minting: 'NFT Related (in work)',
+	Minting: 'Protocols NFT minting Related (in work)',
 	Assets: '(will be removed)',
-	Derivatives: 'Smart contracts that gets its value, risk, and basic term structure from an underlying asset',
-	Payments: 'Offer the ability to pay/send/receive cryptocurrency',
+	Derivatives: 'Protocols that provides smart contracts that gets value, risk, and basic term structure from an underlying asset',
```

Comment:
could be simplified? to protocols for betting with leverage

Reply:
changed!